integration/go_package_rel_proto_in_root/strings.proto

```diff
@@ -1,7 +1,5 @@
 syntax = "proto3";
 
-option go_package="pb/strings";
```

Comment:
```suggestion
option go_package="github.com/utrack/clay/integration/go_package_rel_proto_in_root/pb/strings";
```

Reply:
`option go_package="strings";`

because `option go_package="github.com/utrack/clay/integration/go_package_proto_in_root/pb/strings";
` will create full path in pb/strings folder